presign/base/models.py

```diff
@@ -407,11 +408,29 @@ def change_state(self, action):
         self.state = next_state
         self.save(update_fields=["state"])
 
-    def send_change_state_email(self, request, action):
+        # if the action is not a answer save, we set all current messages to inactive
+        if action != ParticipantStateActions.ANSWERS_SAVED:
+            ParticipantCustomMessage.objects.filter(
+                participant=self, is_current=True
+            ).update(is_current=False)
+
+        if custom_msg:
+            ParticipantCustomMessage.objects.create(
+                participant=self, message=custom_msg, is_current=True
+            )
+
+    def send_change_state_email(self, request, action, custom_msg=None):
         texts = self.event.get_action_email_texts(action)
+        custom_msg_expanded = ""
+        if custom_msg:
+            custom_msg_expanded = (
```

Comment:
why?

Reply:
The idea was to add templating, custom message heading etc. around this here.

This may could have a nicer solution together with i18n, I dont really know yet how to do this nicely.

Reply:
Removed this for now